.github/workflows/actions.yml

```diff
@@ -8,7 +8,7 @@ jobs:
       - name: Create env file
         shell: bash
         run: |
-          echo "export GITHUB_TOKEN=${{ secrets.TOKEN }}" >> .env
+          echo "export GITHUB_TOKEN=${{ secrets.TOKEN }}" >> hydra.cfg
```

Comment:
btw is it a token provided by you or automatically by github? afaik there is `secrets.GITHUB_TOKEN` automatically injected into the action